docker/install.sh

```diff
@@ -14,6 +14,17 @@
 #  License for the specific language governing permissions and limitations under
 #  the License.
 
-source /opt/autoware.ai/ros/install/setup.bash
-cd ~/
-colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release
+# Source ros2
+source /opt/ros/foxy/setup.bash
+# Get driver
+sudo apt-get update && sudo apt install curl gnupg2 lsb-release
+sudo curl -sSL https://raw.githubusercontent.com/ros/rosdistro/master/ros.key  -o /usr/share/keyrings/ros-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/ros-archive-keyring.gpg] http://packages.ros.org/ros2/ubuntu $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/ros2.list > /dev/null
```

Comment:
Is this ros2 key not already in carma-base?

Reply:
I see this key being added in carma base https://github.com/usdot-fhwa-stol/carma-base/blob/a9f0aa253ba9baa825f04bcc9e6354bccf5249bb/Dockerfile#L63
But it seems slightly different